utils/build-script

```diff
@@ -306,311 +306,311 @@ details of the setups of other systems or automated environments.""")
-        help="skip testing all tvOS targets. Equivalent to specifying both "
-             "--skip-test-tvos-simulator and --skip-test-tvos-host",
-        action="store_true")
+                                 help="skip testing all tvOS targets. Equivalent to specifying both "
+                                 "--skip-test-tvos-simulator and --skip-test-tvos-host",
+                                 action="store_true")
     skip_test_group.add_argument("--skip-test-tvos-simulator",
-        help="skip testing tvOS simulator targets",
-        action="store_true")
+                                 help="skip testing tvOS simulator targets",
+                                 action="store_true")
     skip_test_group.add_argument("--skip-test-tvos-host",
-        help="skip testing tvOS device targets on the host machine (the TV "
-             "itself)",
-        action="store_true")
+                                 help="skip testing tvOS device targets on the host machine (the TV "
+                                 "itself)",
+                                 action="store_true")
     skip_test_group.add_argument("--skip-test-watchos",
-        help="skip testing all tvOS targets. Equivalent to specifying both "
-             "--skip-test-watchos-simulator and --skip-test-watchos-host",
-        action="store_true")
+                                 help="skip testing all tvOS targets. Equivalent to specifying both "
+                                 "--skip-test-watchos-simulator and --skip-test-watchos-host",
+                                 action="store_true")
     skip_test_group.add_argument("--skip-test-watchos-simulator",
-        help="skip testing watchOS simulator targets",
-        action="store_true")
+                                 help="skip testing watchOS simulator targets",
+                                 action="store_true")
     skip_test_group.add_argument("--skip-test-watchos-host",
-        help="skip testing watchOS device targets on the host machine (the "
-             "watch itself)",
-        action="store_true")
+                                 help="skip testing watchOS device targets on the host machine (the "
+                                 "watch itself)",
+                                 action="store_true")
 
     parser.add_argument("-i", "--ios",
-        help="""
+                        help="""
 also build for iOS, but disallow tests that require an iOS device""",
-        action="store_true")
+                        action="store_true")
 
     parser.add_argument("--tvos",
-        help="""
+                        help="""
 also build for tvOS, but disallow tests that require a tvos device""",
-        action="store_true")
+                        action="store_true")
 
     parser.add_argument("--watchos",
-        help="""
+                        help="""
 also build for Apple watchos, but disallow tests that require an watchOS device""",
```

Comment:
Unrelated, but we should fix the fact that "watchOS" is spelled two different ways in this single sentence :joy: